### PR TITLE
feat(WebexSettings): add WebexSettings component

### PR DESCRIPTION
**Pull request by patricia0817**
create a settings component with 2 empty tabs

### CODE REVIEW

File: src/components/WebexSettings/WebexSettings.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * @returns {object} JSX of the component
+ */
+
+export default function WebexSettings({data}) {
+  return (
+    <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-settings`}>
+      <Tabs tabType="tabs">
+        <TabList>
+          <Tab
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:27]: make single line where possible:
```html
<Tab heading="Audio"/>

<TabPane>audio settings</TabPane>
```

File: src/components/WebexSettings/WebexSettings.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            heading="Audio"
+          />
+          <Tab
+            heading="Video"
+          />
+        </TabList>
+        <TabContent>
+          <TabPane>
+            {data.audio}
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:36]: the audio and video UIs won't be received via props, they will be implemented right here.

Props to add:
- className
- meetingID

File: src/components/WebexSettings/WebexSettings.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  .md-tab__list {
+    border-bottom: 2px solid $md-gray-20;
+  }
+
+  .md-tab__item.md-type--tabs>a {
+    border: none;
+  }
+
+  .md-tab__item.md-type--tabs {
+   a {
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.scss:11]: indent

File: src/components/WebexSettings/WebexSettings.jsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  TabPane,
+  Tabs,
+} from '@momentum-ui/react';
+import {MeetingContext} from '../hooks';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Settings component
+ *
+ * @param {object} data Data passed to the component
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:17]: double space pls

File: src/components/WebexSettings/WebexSettings.jsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+ * @param {string} props.meetingID  ID of the meeting to control
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:18]: "ID of the meeting" is enough

Comment (cipak) [reply src/components/WebexSettings/WebexSettings.jsx:18]: and add this before the other prop, same order as in the function declaration

File: src/components/WebexSettings/WebexSettings.jsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+ */
+
+export default function WebexSettings({className, meetingID}) {
+  const mainClasses = {
+    [`${WEBEX_COMPONENTS_CLASS_PREFIX}-settings`]: true,
+    [className]: !!className,
+  };
+
+  return (
+    <MeetingContext.Provider value={meetingID}>
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:31]: this meeting context is not needed here

**Comment by cipak** [comment]
please add a basic README.md file

File: src/components/WebexSettings/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Webex Settings Component
+
+Webex Settings component displays a tab menu that allows for control over device selection by performing a specific action.
+
+<p align="center">
+  <img src="./WebexSettings.png" alt="Default Webex Seetings" />
```
Comment (cipak) [inline src/components/WebexSettings/README.md:6]: - this image does not seem to exist. Maybe just add the example image from the story for now.
- Seetings -> Settings
- simpler explanation, for example: Webex Settings component allows the user to choose which camera and microphone to use during the meeting

File: src/components/WebexSettings/WebexSettings.jsx [hunk 8]
```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:4]: is it possible to make this a single line?

File: src/components/WebexSettings/WebexSettings.jsx [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+  TabList,
+  TabPane,
+  Tabs,
+} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Settings component
+ *
+ * @param {string} props.className Custom CSS class to apply
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:16]: - we should put a double space between the arg name and arg description
- better to also have `@param {object} props` at the top of the list

File: src/components/WebexSettings/WebexSettings.jsx [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+ * @param {string} props.meetingID ID of the meeting to control
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:17]: "Meeting id" is enough, or, if you prefer, something like "Id of the meeting to set the preferred devices for"

File: src/components/WebexSettings/WebexSettings.stories.js [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+import WebexSettings from './WebexSettings';
+
+export default {
+  title: 'Meetings/Webex Settings',
+  component: WebexSettings,
+};
+
+const Template = (args) => <WebexSettings {...args} />;
+
+export const Settings = Template.bind({});
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.stories.js:11]: @lalli-flores Known issues:
- momentum tabs don't look good on small mobiles
- we will format the tabs to look like the ones in the image in the user story in a separate task, as it is an application wide feature: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-236060

File: src/components/WebexSettings/WebexSettings.jsx (hunk 8 above)
Comment (patricia0817) [reply src/components/WebexSettings/WebexSettings.jsx:4]: Linter would not allow it

File: src/components/WebexSettings/WebexSettings.jsx [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * Webex Settings component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.className  Custom CSS class to apply
+ * @param {string} props.meetingID  ID of the meeting to set the preferred devices for
+ *
+ * @returns {object} JSX of the component
+ */
+
```
Comment (cipak) [inline src/components/WebexSettings/WebexSettings.jsx:22]: please remove this empty line between jsdoc and function

File: src/components/WebexSettings/WebexSettings.stories.js (hunk 11 above)
Comment (lalli-flores) [reply src/components/WebexSettings/WebexSettings.stories.js:11]: I think we should move away from Momentum, knowing that we will soon not support it

**Pull request merged**